docs/docs/deploying-to-now.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Deploying to Now
+---
+
+In order to deploy your Gatsby project using [Now](https://zeit.co/now), you can do the following:
+
+1.  Install the Now CLI
+
+`npm install -g now`
+
+2.  Install a node server package (such as `serve`, or `http-server`)
+
+`npm install --save serve`
+
+3.  Add a `start` script to your `package.json` file, this is what Now will use to run your application:
+
+`"start": "serve public/"`
```

Comment:
I'd go one level higher in the package.json file to make clear that this should go into the „scripts“ section.